lib/utils.pm

```diff
@@ -296,8 +296,7 @@ sub is_tumbleweed {
 sub is_leap {
     # Leap and its stagings
     return 0 unless check_var('DISTRI', 'opensuse');
-    return 1 if get_var('VERSION', '') =~ /(?:[4-9][0-9]|[0-9]{3,})\.[0-9]/;
-    return get_var('VERSION') =~ /^42:S/;
+    return 1 if get_var('VERSION', '') =~ /(?:[4-9][0-9]|[0-9]{2,})\.[0-9]/;
```

Comment:
Let's simplify to simply use two digits group before dot.

Reply:
done